package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "backbone": "~1.3.2",
     "coffee-script": "1.6.1",
-    "edx-pattern-library": "~0.12.4",
+    "edx-pattern-library": "git+https://github.com/edx/ux-pattern-library.git#f8d8dbcb86c22450566dc6b40db0f23ea0c6fcba",
```

Comment:
FYI, the commit I need is from this PR:

https://github.com/edx/ux-pattern-library/pull/329